modules/gdscript/gdscript_parser.cpp

```diff
@@ -1817,7 +1817,7 @@ GDScriptParser::ForNode *GDScriptParser::parse_for() {
 	n_for->list = parse_expression(false);
 
 	if (!n_for->list) {
-		push_error(R"(Expected a list or range after "in".)");
+		push_error(R"(Expected content after "in".)");
```

Comment:
"content" isn't really descriptive.

Reply:
I want to be descriptive but there are too many things that work, it can't be listed. I can at best say "variant", but it doesn't accept every variant.

- Array = Iterates over indices
- Dictionary = Iterates over keys
- range = You know
- Vector2 and Vector3 = Same as range
- int = range(0, my_int)
- String = Iterates over characters

Also reduz was in support of nodes being taken as a shorthand for `Node.get_children()` so maybe someday we'll have that too.